Kickstarter-iOS/Features/ReportProject/ReportProjectInfoView.swift

```diff
@@ -0,0 +1,99 @@
+import Library
+import SwiftUI
+
+enum ReportProjectHyperLinkType: String, CaseIterable {
+  case prohibitedItems
+  case communityGuidelines
+
+  func stringLiteral() -> String {
+    switch self {
+    case .prohibitedItems:
+      return Strings.Prohibited_items()
+    case .communityGuidelines:
+      return "community guidelines"
```

Comment:
No internationalized text for this from when Android did it?

Reply:
We don't have an internationalized string for this, unfortunately.  This is mainly for the hyperlink. I'm not sure how Android handles hyperlinks, but they wouldn't have an internationalized string. I'll request one from the translations team, but those may not be done by code freeze.

Seems like something that shouldn't hold up this release. What do you think?